fuzzing/nxt_basic_fuzz.c

```diff
@@ -51,19 +51,19 @@ LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 void
 nxt_base64_fuzz(const u_char *data, size_t size)
 {
-    u_char  *input;
-    u_char  buf[256];
+    ssize_t  ret;
+    u_char   buf[256];
```

Comment:
In Unit we generally order out variables in 'Christmas tree' order of the variable type (then alphabetically).

So these two should be switched around...

Reply:
> Large arrays are put at the end of a declaration block. [code_style_variables](https://nginx.org/en/docs/dev/development_guide.html#code_style_variables)

Ohh, I thought arrays in general are put at the end of a declaration block.

Reply:
Hmm, we do deviate from the nginx coding style in some places and I do struggle to keep up with all the oddities we have.

Personally I'm not fussed in what order variables are declared. But here's some corroborating evidence...

```c
nxt_int_t
nxt_cgroup_proc_add(nxt_task_t *task, nxt_process_t *process)
{
    int        len;
    char       cgprocs[NXT_MAX_PATH_LEN];
    FILE       *fp;
    nxt_int_t  ret;
```